src/App.tsx

```diff
@@ -1,20 +1,17 @@
-// import logo from './logo.svg';
 import './App.css';
 import { Route, Routes } from 'react-router';
-import { ROUTE_PATH_ArchivePage, ROUTE_PATH_LandingPage, ROUTE_PATH_Mockman, ROUTE_PATH_TrashPage } from './utils/Route';
-// import HomePage from "./pages/HomePage/HomePage";
-// import HomePage from './pages/HomePage/HomePage';
-// import { TrashPage } from "./pages/HomePage/TrashPage";
-// import { ArchivePage } from "./pages/HomePage/ArchivePage";
+import { ROUTE_PATH_ArchivePage, ROUTE_PATH_LabelPage, ROUTE_PATH_LandingPage, ROUTE_PATH_Mockman, ROUTE_PATH_TrashPage } from './utils/Route';
```

Comment:
Variable namings can be better. Also, avoid mixing uppercase and lowerCase together

Reply:
done